### ISSUE TITLE
Error during processing ReadAllEventsForward request.

### ISSUE THREAD

**Issue by ianbattersby** [EventStore/EventStore#694]
**Version: v3.2.1**

```
[16428,54,15:37:56.418] Error during processing ReadAllEventsForward request. 
Log record at actual pos 87311 has too large length: 1148478823 bytes, while limit is 16777216 bytes. In chunk #0-0 (chunk-000000.000000).
```

Received this on Windows when enabling a projection, yet it doesn't show as Faulted. Anything to worry about?

**Comment by ianbattersby** [comment]
Apologies, I think this is related to this which might be resolved in v3.3.0: https://github.com/EventStore/EventStore.UI/issues/85

**Comment by jen20** [comment]
Normally this means the server has received a nonsense position as the start of the read all operation (i.e. into the middle of a record). That fix is actually in 3.2.0 (#653).
